warehouse/query-core/src/main/java/datawave/query/jexl/visitors/ExpandMultiNormalizedTerms.java

```diff
@@ -259,6 +261,14 @@ protected JexlNode expandNodeForNormalizers(JexlNode node, Object data) {
                             if (log.isTraceEnabled()) {
                                 log.trace("Could not normalize " + term + " using " + normalizer.getClass());
                             }
+                            // if this was a regex, then lets assume that the regex could not be decoded enough to determine how to match against the index.
+                            // in this case we need to force this term to be evaluation only (i.e. cannot be done against an index)
+                            if (node instanceof ASTNRNode || node instanceof ASTERNode) {
```

Comment:
Can we expand the test case below to also hit the ASTNRNode?

Reply:
I did, and I also realized that we were not handling them correctly.  Note that the NR and NE cases are not actually ever exercised  because all NE and NR nodes are turned into EQ and ER nodes surrounded by a negation before we get here.